tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -76,11 +55,19 @@ def get_configuration(request):
     return request.param
 
 
-@pytest.mark.parametrize('key_string, key_object, tags_to_apply', [
-    (keys_strings, keys_objects, {'ignore_registry'})
+@pytest.mark.parametrize('root_key, registry, arch, subkey, triggers_event, tags_to_apply', [
+    (key, subkey_1, KEY_WOW64_32KEY, "some_key", True, {'ignore_registry_key'}),
+    (key, subkey_1, KEY_WOW64_64KEY, "some_key", True, {'ignore_registry_key'}),
+    (key, subkey_1, KEY_WOW64_64KEY, ignore_key, False, {'ignore_registry_key'}),
+    (key, subkey_1, KEY_WOW64_32KEY, ignore_key, False, {'ignore_registry_key'}),
+    (key, subkey_1, KEY_WOW64_64KEY, "regex_ignored_key", False, {'ignore_registry_key'}),
+    (key, subkey_1, KEY_WOW64_32KEY, "regex_ignored_key", False, {'ignore_registry_key'}),
+    (key, subkey_2, KEY_WOW64_64KEY, "some_key", True, {'ignore_registry_key'}),
+    (key, subkey_2, KEY_WOW64_64KEY, ignore_key, False, {'ignore_registry_key'}),
+    (key, subkey_2, KEY_WOW64_64KEY, "regex_ignored_key", False, {'ignore_registry_key'})
 ])
-def test_ignore_registry(key_string, key_object, tags_to_apply, get_configuration,
-                         configure_environment, restart_syscheckd, wait_for_initial_scan):
+def test_ignore_registry_key(root_key, registry, subkey, arch, triggers_event, tags_to_apply,
+                         get_configuration, configure_environment, restart_syscheckd, wait_for_fim_start):
```

Comment:
This line is under-indented.
```suggestion
                             get_configuration, configure_environment, restart_syscheckd, wait_for_fim_start):
```